examples/platformio/src/main.cpp

```diff
@@ -19,12 +19,17 @@
 #include <Sensors.hpp>
 
 void onSensorDataOk() {
-    Serial.print ("-->[MAIN] PM1.0: "+sensors.getStringPM1());
+    Serial.print ("-->[MAIN] PM1: "+sensors.getStringPM1());
     Serial.print (" PM2.5: " + sensors.getStringPM25());
     Serial.print (" PM10: " + sensors.getStringPM10());
-    Serial.print (" PM1: " + sensors.getStringPM1());
-    Serial.print (" T: " + String(sensors.getTemperature()));
-    Serial.println(" H: "+ String(sensors.getHumidity()));
+    
+ //   Serial.print (" CO2: " + sensors.getStringCO2());
+ //   Serial.print (" CO2humi: " + String(sensors.getCO2humi()));
+ //   Serial.print (" CO2temp: " + String(sensors.getCO2temp()));
```

Comment:
these comments is for? maybe we could remove it in this example.

Reply:
When I uncomment this lines the Platformio online shows errors:
src/main.cpp:26:38: error: 'class Sensors' has no member named 'getStringCO2'

In my computer the build is OK, I don´t know the reason
I can´t find the solution, I leave commented meanwhile